### ISSUE TITLE
False positive property-with-parameters for an @abstractproperty setter

### ISSUE THREAD

**Issue by volans-** [PyCQA/pylint#3600]
### Steps to reproduce
1. Define the following code with an abstract setter:
```python
"""MyModule."""
from abc import ABC, abstractproperty


class MyClassBase(ABC):
    """MyClassBase."""

    @abstractproperty
    def example(self):
        """Getter."""

    @abstractproperty
    @example.setter
    def example(self, value):
        """Setter."""
```
2. Run pylint on the file

### Current behavior
The following problem is reported for the abstract setter:
```
example.py:14:4: R0206: Cannot have defined parameters for properties (property-with-parameters)
```
Removing the `@abstractproperty` on line 12 makes the `property-with-parameters` problem disappear.

### Expected behavior
It should not report any problem for the definition of the abstract setter as it's correct that it has a parameter to set the value.

### pylint --version output
```console
$ pylint --version
pylint 2.5.2
astroid 2.4.1
Python 3.7.7 (default, Mar 10 2020, 15:43:03)
[Clang 11.0.0 (clang-1100.0.33.17)]
```
